File: Programmers/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
	vector<int> answer;
	for (int i = 0; i<commands.size(); i++) {
		vector<int> tmp = array;
		int a = commands[i][0];
		int b = commands[i][1];
		int c = commands[i][2];
		sort(tmp.begin() + a - 1, tmp.begin() + b);
		answer.push_back(*(tmp.begin() + a + c - 2));
	}
	return answer;
}

File: Programmers/K번째수.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int n = commands.length;
        int[] answer = new int[n];
        for (int i = 0; i < n; i++) {
            int a = commands[i][0];
            int b = commands[i][1];
            int c = commands[i][2];
            int[] temp = Arrays.copyOfRange(array, a - 1, b);
            Arrays.sort(temp);
            answer[i] = temp[c - 1];
        }
        return answer;
    }
}